op-acceptor/logging/filelogger_test.go

```diff
@@ -714,93 +714,6 @@ func TestHTMLSummarySink_WithSubtestsAndNetworkInfo(t *testing.T) {
 // TestExtractErrorInfoFromJSON verifies that error information is correctly extracted from test output
 func TestExtractErrorInfoFromJSON(t *testing.T) {
```

Comment:
This seems like it left an empty test function behind

Reply:
Oh and the `extractErrorData` is still there - we sure we want to remove the tests?

Reply:
Good spot, it's indeed still used, so I'll restore this test. Cheers